web/src/components/TaskDetailModal.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import TaskContent from "./TaskContent";
+import Button from "@leafygreen-ui/button";
+import Modal from "@leafygreen-ui/modal";
+
+import useTaskMutations from "../graphql/useTaskMutations";
+
+export default function TaskDetailModal({ project, task, unselectTask }) {
+  const TaskStatusButton = useTaskStatusButton(project);
+  return (
+    <Modal open={Boolean(task)} setOpen={unselectTask}>
```

Comment:
Super simple way to close/open the modal! Might be worth adding a comment or mentioning in the tutorial why you're doing "Boolean(task)" to make it more easily understandable to users.